en/mkdocs.yml

```diff
@@ -106,6 +106,14 @@ nav:
           - Programmatic Access to Choreo Insights API: observe-and-analyze/analyze/programmatic-access-choreo-insights-api.md
   - Alerts: alerts.md
   - DevOps: devops/devops-portal.md
+  - iPaaS:
+      - Getting Started: ipaas/getting-started.md
+      - Ballerina:
+          - Developing Integrations with Ballerina: ipaas/ballerina/developing-integrations-with-ballerina.md
+          - Create Your First Ballerina Event-Triggered Integration: ipaas/ballerina/create-your-first-ballerina-event-triggered-integration.md
```

Comment:
Change "- Create Your First Ballerina Event-Triggered Integration: ipaas/ballerina/create-your-first-ballerina-event-triggered-integration.md" to "- Create Your First Event-Triggered Integration: ipaas/ballerina/create-your-first-event-triggered-integration.md